b2share/modules/records/permissions.py

```diff
@@ -30,10 +30,13 @@
 
 from invenio_access.permissions import superuser_access, \
     ParameterizedActionNeed
```

Comment:
Check if there is cyclic dependency problems in this file and if none are found the imports from methods can be removed